app/src/controllers/user.js

```diff
@@ -144,6 +159,19 @@ module.exports = {
     callback(client);
```

Comment:
I think you are missing a call to this.stopPinging() here like you have in the rejectGroupName.